aten/src/ATen/native/quantized/cpu/qembeddingbag_prepack.cpp

```diff
@@ -94,7 +94,7 @@ namespace {
 // Note - This is a temporary pack function for embedding bag which quantizes
 // and packs the float weight tensor. In the next step it will be replaced by a
 // quantize and pack function once we support FP scale and FP zero_point
-Tensor qembeddingbag_byte_prepack(const Tensor& weight) {
+Tensor qembeddingbag_byte_prepack(const Tensor& weight, bool optimized_qparams) {
```

Comment:
This function is not doing optimized_qparam calculation, can you please add that?

Reply:
In caffe2 we currently don't have optimized_qparam calculation for `byte_prepack`. Do we need it in PT?